currency-v1/apiproxy/resources/jsc/ConfigureTargetRequest.js

```diff
@@ -1,3 +1,3 @@
 context.setVariable('target.copy.pathsuffix', false);
 
-context.setVariable('bella', 'cool');
+context.setVariable('Geir', 'cool');
```

Comment:
what is the point of this? it doesnt do anything, but it is Thursday afternoon and we need to release so im merging anyway